notebooks/TFSegFormer_ONNX.ipynb

```diff
@@ -1,1418 +1,1515 @@
 {
```

Comment:
<ul><li>maybe leave a note about how the speed is measured? </li><li>also, it would be nice to show the average time besides the total amount of time. </li><li>one more thing. Do we really need warmup here? Since the model is fully loaded up from the previous cells, I am not sure if warmup makes any difference.</li></ul><p> </p><p>WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/segformer-tf-transformers/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I will add the additional details. 

Model warm-up is a safe bet. In all the benchmarking works I have seen they always perform warmup.